components/dropbox/sources/new-file/new-file.mjs

```diff
@@ -1,15 +1,27 @@
-const dropbox = require("../../dropbox.app.js");
+import dropbox from "../../dropbox.app.mjs";
 
-module.exports = {
+export default {
   key: "dropbox-new-file",
   name: "New File",
-  version: "0.0.4",
-  description:
-    "Emits an event when a new file is added to your account or a specific folder. Make sure the number of files/folders in the watched folder does not exceed 4000.",
+  version: "0.0.5",
+  description: "Emits an event when a new file is added to your account or a specific folder. Make sure the number of files/folders in the watched folder does not exceed 4000.",
   props: {
     dropbox,
-    path: { propDefinition: [dropbox, "path"] },
-    recursive: { propDefinition: [dropbox, "recursive"] },
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFolder",
+        () => ({
+          returnSimpleString: true,
+        }),
+      ],
+    },
+    recursive: {
+      propDefinition: [
+        dropbox,
+        "recursive",
+      ],
+    },
```

Comment:
If you create a `common.mjs` file in sources, include `...common.props` in the props section and remove `dropbox`, `db`, `path`, `recursive` & `dropboxApphook`

Also, `includeMediaInfo` and `includeLink` need labels.